tool/create-exercise

```diff
@@ -75,6 +81,86 @@ dev_dependencies:
   test: '0.12.23+1'
 """;
 
+String testCaseTemplate(name, testCase) {
+  if (testCase['cases'] != null) {
+    // We have a group, not a case
+    String tests = testCase['cases'].map((c) => testCaseTemplate(name, c)).join("\n");
+    String description = testCase['description'];
+
+    if (description == null) {
+      return tests;
+    }
+
+    return """
+      group("$description", () {
+        $tests
+      });
+    """;
+  }
+
+  String description = repr(testCase['description']);
+  String resultType = getFriendlyType(testCase['expected']);
+  String object = camelCase(name);
+  String method = camelCase(testCase['property']);
+  String expected = repr(testCase['expected']);
+
+  List<String> inputKeys = testCase.keys.toList()..remove("expected")..remove("description")..remove("property");
+  String arguments = inputKeys.map((k) => repr(testCase[k])).join(", ");
+
+  return """
+    test($description, () {
+      final $resultType result = $object.$method($arguments);
+      expect(result, equals($expected));
+    }, skip: true);
+  """;
+}
+
+/// `repr` takes in any object and tries to coerce it to a String in such a way that it is suitable to include in code.
+/// Based on the python `repr` function, but only works for basic types: String, Iterable, Map, and primitive types
+String repr(dynamic x) {
```

Comment:
Following the [Design Guide on dartlang](https://www.dartlang.org/guides/language/effective-dart/design#do-annotate-with-object-instead-of-dynamic-to-indicate-any-object-is-accepted), recommend using `Object` instead of dynamic to denote any object is accepted. Do this for the other parameters which have dynamic that you added as well.